components/org.wso2.carbon.deployment.notifier/src/test/resources/yaml/conf/deployment.andes.yaml

```diff
@@ -0,0 +1,58 @@
+################################################################################
+#   Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+################################################################################
+
+# Deployment Engine related configurations
+#deployment:
+
+# Currently there can be two modes
+#  1. Scheduled Mode - (enable below property to "scheduled") - where the task runs periodically and trigger deployment
+#  2. Triggered Mode - (enable below property to "triggered") - the deployment has to be triggered externally,
+#   eg : in a worker node we don't need the task to run, but rather when we receive a
+#        cluster msg,  the deployment has to be triggered manually
+wso2.deployment:
```

Comment:
Let's change the namespace as 'wso2-artifact-deployment' to better reflect its functionality. Otherwise, this conflicts with the deployment.yaml that represents the config file for the entirety of the product's deployment.